test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -763,23 +769,25 @@ def test_cgroup_cpuset(self):
         """
         name = "CGROUP4"
         self.load_config(self.cfg3)
+        # Restart MoM to work around PP-993
+        self.mom.restart()
         a = {'Resource_List.select': '1:ncpus=1:mem=300mb',
              ATTR_N: name + 'a'}
         j1 = Job(TEST_USER, attrs=a)
         j1.create_script(self.cpuset_mem_script)
         jid1 = self.server.submit(j1)
+        a = {'job_state': 'R'}
+        self.server.expect(JOB, a, jid1)
+        self.server.status(JOB, [ATTR_o, ATTR_e], jid1)
```

Comment:
I don't see you using the error file.  If so, why query it?

Reply:
Not sure. Removed.